statistics/handle/cache/internal/lfu/lfu_cache.go

```diff
@@ -130,3 +148,9 @@ func (s *LFU) wait() {
 func (s *LFU) metrics() *ristretto.Metrics {
 	return s.cache.Metrics
 }
+
+// Stop implements statsCacheInner
+func (s *LFU) Stop() {
```

Comment:
How about renaming it from `Stop()` to `Close()` to keep aligned with the underlying `s.cache.Close()`?

Reply:
Done